packages/reports/src/reportBuilder/fetch/types.ts

```diff
@@ -0,0 +1,5 @@
+import { Row } from '../types';
+
+export interface FetchResponse {
+  results: Row[];
+}
```

Comment:
Separating types in different files is a new pattern which makes a lot of sense! 👌

Reply:
Yeah, I agree. I did it to avoid dependency cycles, but honestly it just makes things a lot easier to read.

I am a fan tho of having multiple different `types.ts` files within various packages. I'm not entirely sure why, but my code feels more appropriately encapsulated that way.